htdocs/install/mysql/migration/16.0.0-17.0.0.sql

```diff
@@ -144,3 +144,5 @@ CREATE TABLE llx_bank_extrafields
 ALTER TABLE llx_bank_extrafields ADD INDEX idx_bank_extrafields (fk_object);
 
 ALTER TABLE llx_user CHANGE COLUMN note note_private text;
+
+UPDATE llx_c_effectif SET code='EF101-500', libelle='101 - 500' WHERE id=5;
```

Comment:
UPDATE llx_c_effectif SET code='EF101-500', libelle='101 - 500' WHERE code='EF100-500'
seems more secured.

Reply:
I'm not very used to git, and I don't know how how to modify a pull request.
So after some attempts, I pushed another pull request taking account your comment.

Reply:
Hello,
I managed to make the changes here
https://github.com/Dolibarr/dolibarr/pull/22076/files